diff_tf:
  ros__parameters:
    rate: 10
    ticks_meter: 320
    base_width: 0.245
    base_frame_id: base_footprint
    odom_frame_id: odom
    encoder_min: -32768
    encoder_max: 32768
    radius: 0.033
    left_wheel_frames:
      - wheel_front_left_link
      - wheel_back_left_link
    right_wheel_frames: 
      - wheel_front_right_link
      - wheel_back_right_link
    # wheel_low_wrap 
    # wheel_high_wrap

    # wheels:
    #   separation: 0.287
    #   radius: 0.033

    # motors:
    #   profile_acceleration_constant: 214.577
    # profile_acceleration: 0.0

    odometry:
      publish_tf: true
      publish_joints: true
      use_imu: true
      frame_id: "odom"
      child_frame_id: "base_footprint"

